reasoner_util/__init__.py

```diff
@@ -185,3 +185,34 @@ def merge_qnodes(qnodes1: dict, qnodes2: dict) -> dict:
         elif qnode_value != new_nodes[qnode_key]:
             raise ValueError("Key exists in both messages but values do not match.")
     return new_nodes
+
+
+def merge_attributes(knode1_attributes: List[dict], knode2_attributes: List[dict]) -> List[dict]:
+    """Find the union of the attributes lists in knowledge graph nodes that
+    require merging."""
+    new_node_attributes = copy.deepcopy(knode1_attributes)
+    for attribute2 in knode2_attributes:
+        if attribute2 not in new_node_attributes:
+            new_node_attributes += copy.deepcopy(attribute2)
```

Comment:
I think this should be 
```
new_node_attributes += [copy.deepcopy(attribute2)]
```

or 

```
new_node_attributes.append(attribute2)
```

Since `attribute2` will be a `Dict` while `new_node_attributes` will be a `List`